pyproject.toml

```diff
@@ -13,6 +13,8 @@ dependencies = [
     # tomli is only required below 3.11 as it is native after that version.
     'tomli; python_version<"3.11"',
     "requests",
+    "dasbus==1.7",
+    "PyGObject==3.50.0",
```

Comment:
Do these need to be locked to specific version? It's generally better to avoid being so specific about dependencies unless there is a known incompatibility.

Reply:
As far as I know, they don't require any specifics

Reply:
Let's avoid pinning these unless we need to.